tests/__snapshots__/index.test.ts.snap

```diff
@@ -46,12 +46,12 @@ type doc
 `;
 
 exports[`friendlySyntaxToApiSyntax() should be able to handle single character type 1`] = `
-Object {
-  "type_definitions": Array [
-    Object {
-      "relations": Object {
-        "r": Object {
-          "this": Object {},
+{
```

Comment:
do we change this on purpose?  Was there product change?

Reply:
This was a jest update, no need for Object and Array to be set